projects/angular/src/emphasis/alert/_alert.clarity.scss

```diff
@@ -11,272 +11,138 @@
-    alert-variables.getAlertColor($alertType, color),
-    variables.$clr-use-custom-properties
-  );
+  background: alert-variables.getAlertColor($alertType, bg-color);
+  color: alert-variables.getAlertColor($alertType, color);
 
   & a {
-    @include mixins.css-var(
-      color,
-      alert-variables.getAlertColor($alertType, color, 'css-var'),
-      alert-variables.getAlertColor($alertType, color),
-      variables.$clr-use-custom-properties
-    );
+    color: alert-variables.getAlertColor($alertType, color);
   }
 
   $border-prop: alert-variables.getAlertColor($alertType, border-color);
-
   @if $border-prop != 'none' {
-    border: alert-variables.$clr-alert-borderwidth solid;
-    @include mixins.css-var(
-      border-color,
-      alert-variables.getAlertColor($alertType, border-color, 'css-var'),
-      $border-prop,
-      variables.$clr-use-custom-properties
-    );
+    border: alert-variables.$clr-alert-borderwidth solid $border-prop;
   } @else {
     border: none;
   }
 
   .alert-icon {
-    @include mixins.css-var(
-      color,
-      alert-variables.getAlertColor($alertType, icon, 'css-var'),
-      alert-variables.getAlertColor($alertType, icon),
-      variables.$clr-use-custom-properties
-    );
+    color: alert-variables.getAlertColor($alertType, icon);
   }
 
-  $myActionCssVar: alert-variables.getAlertColor($alertType, action, 'css-var');
   $myActionSassVar: alert-variables.getAlertColor($alertType, action);
-  $myActionActiveCssVar: alert-variables.getAlertColor($alertType, action-active, 'css-var');
-  $myActionActiveSassVar: alert-variables.getAlertColor($alertType, action-active);
 
   .alert-actions .dropdown .dropdown-toggle {
-    @include mixins.css-var(color, $myActionCssVar, $myActionSassVar, variables.$clr-use-custom-properties);
-    @include mixins.css-var(border-color, $myActionCssVar, $myActionSassVar, variables.$clr-use-custom-properties);
+    color: $myActionSassVar;
+    border-color: alert-variables.getAlertColor($alertType, action-active);
   }
 
   .alert-action,
   .dropdown-toggle {
-    @include mixins.css-var(color, $myActionCssVar, $myActionSassVar, variables.$clr-use-custom-properties);
+    color: $myActionSassVar;
 
     &.btn {
-      @include mixins.css-var(border-color, $myActionCssVar, $myActionSassVar, variables.$clr-use-custom-properties);
-      @include mixins.css-var(background-color, clr-btn-transparent-bg-color, 0, variables.$clr-use-custom-properties);
+      border-color: $myActionSassVar;
+      color: $myActionSassVar;
 
       cds-icon,
       clr-icon {
-        @include mixins.css-var(color, $myActionCssVar, $myActionSassVar, variables.$clr-use-custom-properties);
-      }
-
-      &:focus,
-      &:hover,
-      &:active {
-        @include mixins.css-var(
-          border-color,
-          $myActionActiveCssVar,
-          $myActionActiveSassVar,
-          variables.$clr-use-custom-properties
-        );
+        color: $myActionSassVar;
```

Comment:
Shouldn't this be different for 'focus,hover,active' ?

Reply:
Alert action buttons will follow `outline button` specification. 

Active is equal to regular `button outline` colors.
Hover is handled by `@extend .btn-{alertType}-outline;` lines in  `.alert-app-level { ... }` section.